php deprecated creation of dynamic property warning

### Output

```
─➤  phpbrew list
PHP Deprecated:  Creation of dynamic property PhpBrew\Console::$loader is deprecated in phar:///usr/local/Cellar/phpbrew/2.0.0/bin/phpbrew/vendor/corneltek/cliframework/src/Application.php on line 117

Deprecated: Creation of dynamic property PhpBrew\Console::$loader is deprecated in phar:///usr/local/Cellar/phpbrew/2.0.0/bin/phpbrew/vendor/corneltek/cliframework/src/Application.php on line 117
PHP Deprecated:  Creation of dynamic property CLIFramework\ArgInfo::$desc is deprecated in phar:///usr/local/Cellar/phpbrew/2.0.0/bin/phpbrew/vendor/corneltek/cliframework/src/ArgInfo.php on line 38

Deprecated: Creation of dynamic property CLIFramework\ArgInfo::$desc is deprecated in phar:///usr/local/Cellar/phpbrew/2.0.0/bin/phpbrew/vendor/corneltek/cliframework/src/ArgInfo.php on line 38
PHP Deprecated:  Creation of dynamic property CLIFramework\ArgInfo::$desc is deprecated in phar:///usr/local/Cellar/phpbrew/2.0.0/bin/phpbrew/vendor/corneltek/cliframework/src/ArgInfo.php on line 38

Deprecated: Creation of dynamic property CLIFramework\ArgInfo::$desc is deprecated in phar:///usr/local/Cellar/phpbrew/2.0.0/bin/phpbrew/vendor/corneltek/cliframework/src/ArgInfo.php on line 38
* php-8.1.17
```


### Expected Result

```
actual list
```

### Command

Paste your phpbrew install command here (be sure to install with `--debug` flag, e.g. `phpbrew --debug install`)

    phpbrew list

### Build Log

    trying to run any phpbrew command i get the above code.  When doing any install there is also the same php deprecated messages.  I can't do a `brew install phpbrew@1.28` or anything other than the latest.

### Platform

OS:

    ─➤  sw_vers
    ProductName:            macOS
    ProductVersion:         13.2.1
    BuildVersion:           22D68

Running PHP: 
    ─➤  php -v
    PHP 8.2.4 (cli) (built: Mar 16 2023 16:46:52) (NTS)
    Copyright (c) The PHP Group
    Zend Engine v4.2.4, Copyright (c) Zend Technologies
        with Zend OPcache v8.2.4, Copyright (c), by Zend Technologies

Installing PHP:
    installing any version

i just manually downloaded phpbrew 2.1.0 and I did the `phpbrew init` and I got the following message...

```
Box Requirements Checker
========================

> Using PHP 8.2.5
> PHP is using the following php.ini file:
  /usr/local/etc/php/8.2/php.ini

> Checking Box requirements:
  E.


 [ERROR] Your system is not ready to run the application.


Fix the following mandatory requirements:
=========================================

 * The application requires the version "^7.2.5 || <8.2.0" or greater.
```

Not sure what this really means, when its saying essentially "less than php version 8.2 or greater".  
I guess I might need to downgrade my version of PHP?  Or wait until phpbrew catches up with the version of PHP 8.2.5?

--- a/.github/workflows/tests.yaml
+++ b/.github/workflows/tests.yaml
@@ -19,6 +19,8 @@ jobs:
                     - '7.4'
                     - '8.0'
                     - '8.1'
+                    - '8.2'
+                    - '8.3'
         name: PHP ${{ matrix.php-versions }} Test on ${{ matrix.operating-system }}
 
         steps:

--- a/composer.json
+++ b/composer.json
@@ -24,12 +24,12 @@
     "minimum-stability": "dev",
     "prefer-stable": true,
     "require": {
-        "php": "^7.2.5 || <8.2.0",
+        "php": "^7.2.5 || ^8.0",
         "corneltek/pearx": "^1.3.5",
         "corneltek/curlkit": "^1.0.11",
         "ext-simplexml": "*",
         "symfony/yaml": "^5.4",
-        "corneltek/cliframework": "3.0.x-dev#01cee2d491085cfae63c677611b5e4604624771e"
+        "corneltek/cliframework": "3.0.x-dev#0e69e4d9d8ee6c8b5abb7ebf0cf846e47d93b656"
     },
     "require-dev": {
         "php-vcr/php-vcr": "^1.5.4",

